daml/Marketplace/Custody/Service.daml

```diff
@@ -190,4 +117,11 @@ template Request
         operator : Party
       controller operator, provider
       do
+        let account = createAccount provider customer
         createOrLookup Service with ..
+
+createAccount : Party -> Party -> Account
+createAccount provider customer = do
+  let id = Id with signatories = fromList [customer, provider]; label = partyToText customer <> "@" <> partyToText provider; version = 0
+      owner = customer
```

Comment:
Comment for other reviewers: this is - for now - the default account naming we agreed upon. We can consider re-adding user-defined account naming (or in fact multiple account support) later again, if we see the need for it.